### PR TITLE
Fix bug 1384385: Handle FxA primaryEmailChanged messages

### PR DESCRIPTION
**Pull request by pmac**

### CODE REVIEW

**Comment by pmac** [comment]
cc @rfk

File: basket/news/tasks.py [hunk 1]
```diff
@@ -212,6 +212,24 @@ def fxa_source_url(metrics):
     return source_url
 
 
+@et_task
+def fxa_email_changed(data):
+    ts = data['ts']
```
Comment (glogiotatidis) [inline basket/news/tasks.py]: q: Is `data['ts']` type `float`?

Comment (rfk) [reply basket/news/tasks.py]: Yes, it should be a float in seconds; generated here for reference:

  https://github.com/mozilla/fxa-auth-server/blob/d205c9abee380e250413570b9873561f55bb5068/lib/log.js#L128

**Comment by pmac** [comment]
@rfk so in your opinion is this ready to roll out on my side? are these messages coming through the queue now, or they will be soon? I don't think timing is important on this one right?

**Comment by rfk** [comment]
Yep, should be ready to go with no extra deploy dependencies

**Comment by pmac** [comment]
Fantastic. Many thanks to you both!

**Pull request merged**